lib/index.js

```diff
@@ -35,7 +37,7 @@ module.exports = config => {
   router.get('/download', routes.download)
   router.get('/download/:platform', routes.downloadPlatform)
   router.get('/update/:platform/:version', routes.update)
-  router.get('/update/win32/:version/RELEASES', routes.releases)
+  // router.get('/update/win32/:version/RELEASES', routes.releases)
```

Comment:
We'll presumably eventually need to support this endpoint for WIndows updates but I wanted to have a Windows machine around to test with before poking around here, so I commented this out for now.